server/application/application.go

```diff
@@ -1988,13 +1990,19 @@ func (s *Server) getUnstructuredLiveResourceOrApp(ctx context.Context, rbacReque
 		if err != nil {
 			return nil, nil, nil, nil, fmt.Errorf("error getting application cluster config: %w", err)
 		}
+		data, _ := json.Marshal(obj)
+		fmt.Print("************************************ app " + bytes.NewBuffer(data).String() + "*************************************")
+
 		obj, err = kube.ToUnstructured(app)
 	} else {
 		res, config, app, err = s.getAppLiveResource(ctx, rbacRequest, q)
 		if err != nil {
 			return nil, nil, nil, nil, fmt.Errorf("error getting app live resource: %w", err)
 		}
 		obj, err = s.kubectl.GetResource(ctx, config, res.GroupKindVersion(), res.Name, res.Namespace)
+		data, _ := json.Marshal(obj)
```

Comment:
We should probably handle the error here.

Reply:
Was temporarily used use for debugging printouts, now gone.